recipes/gargammel-lite/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "1.1.2" %}
+{% set sha256_gargammel = "c4b5ca081087fa5ecf086dbd49a4db0fb3c7d2d3d13eb30bdfc96b964661c3c1" %}
+{% set sha256_libgab = "ef4d4e980918f48b9a5c454f46afe3ffe6d97a075bbbf89abb7e3553cd6b9827" %}
+
+package:
+  name: gargammel-lite
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: False  # [osx]
+
+source:
+  - url: https://github.com/grenaud/libgab/archive/d2bbab7d88fbfdaa0ce57a26dc908d2fc66bf656.tar.gz
+    sha256: {{ sha256_libgab }}
+    folder: gargammel/libgab
+  - url: https://github.com/grenaud/gargammel/archive/{{ version }}.tar.gz
+    sha256: {{ sha256_gargammel }}
+    folder: gargammel
+    patches:
+      - 0001-gargammel-Makefile-keep-basic.patch
```

Comment:
Can you make sure those patches get upstreamed, please?

Reply:
I made some of the changes to pass the circleCI test but no idea if this can be applied upstream as there is a lot of hardcoded stuff.